test/lit.cfg

```diff
@@ -501,8 +501,6 @@ backtrace_on_crash = lit_config.params.get('backtrace_on_crash', None)
 if backtrace_on_crash is not None:
     config.environment['SWIFT_BACKTRACE'] = 'enable=on'
 
-config.available_features.add('lld_lto')
```

Comment:
This was used to avoid issues on Android by @kateinoigakukun in #32430, but I enabled it for Android too when it switched to lls